corehq/apps/data_interfaces/tests/tests.py

```diff
@@ -15,22 +17,30 @@
 THISDIR = dirname(abspath(__file__))
 BASE_PATH = join(THISDIR, 'files')
 BASIC_XLSX = 'basic_forms_bulk.xlsx'
+DOMAIN_NAME = 'test'
 MISSING_XLSX = 'missing_forms_bulk.xlsx'
 MALFORM_XLSX = 'malformatted_forms_bulk.xlsx'
 WRONG_FILETYPE = 'wrong_file.xyz'
 
 
+def setup():
+    create_domain(DOMAIN_NAME)
+
+
+def teardown():
```

Comment:
It's a feature of `nose` [*test modules*](http://nose.readthedocs.org/en/latest/writing_tests.html#test-modules)